.github/workflows/build_cmake.yml

```
@@ -0,0 +1,37 @@
+name: CMake Build
+
+on: [ push, pull_request ]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ windows-latest, ubuntu-latest, macos-latest ]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Prerequisites
+        run: cmake -P ${{ github.workspace }}/.github/workflows/install_prerequisites.cmake
```

Can't you simply do
```suggestion
        run: cmake -P .github/workflows/install_prerequisites.cmake
```
?

---

Probably works, though my experience with CI runners is that you should never, ever rely on the working dir being set to something specific, as some runners will not reset it after the previous action. It's simply more safe to always use absolute paths.